buyer-api/src/blockchain/balance.js

```diff
@@ -14,7 +14,7 @@ export const getWeiBalance = async address =>
   toBN(await web3.eth.getBalance(address));
 export const getWibBalance = async address =>
   toBN(await Wibcoin.methods.balanceOf(address).call());
-export const getBatPayBalance = async ({ id }) =>
+export const getBatPayBalance = async (id) =>
```

Comment:
```suggestion
export const getBatPayBalance = async id =>
```
Change this to have lint working.